src/sentry/web/urls.py

```diff
@@ -355,7 +355,7 @@
         r"^accept/(?P<member_id>\d+)/(?P<token>\w+)/$",
         GenericReactPageView.as_view(auth_required=False),
         name="sentry-accept-invite",
-    ),
+    ),  # TODO: does this need to be converted as well?
```

Comment:
Yes. Without an organization slug here we won't be able to send the user to the correct region. Longer term this will end up without an org slug in the path, and with the org slug in the subdomain.